src/main/java/com/softserve/sportshub/article/command/UpdateArticleCategoryCommand.java

```diff
@@ -0,0 +1,16 @@
+package com.softserve.sportshub.article.command;
```

Comment:
I'd like to see this class in the same package as the other command or the other way around

Reply:
You are right @PawelMiskowiec , I'm on it!